rasa/core/policies/rule_policy.py

```diff
@@ -530,12 +550,17 @@ def _check_prediction(
             return []
 
         tracker_type = "rule" if tracker.is_rule_tracker else "story"
+        if tracker_type == "story" and predicted_action_name == ACTION_LISTEN_NAME:
```

Comment:
```suggestion
        if not tracker.is_rule_tracker and predicted_action_name == ACTION_LISTEN_NAME:
```

Reply:
we need to check that previous action before this action_listen was unpredictable